### DISCUSSION TITLE
Restablecer api

### DISCUSSION THREAD

**Discussion by Mariaperez00** [community/community#146718] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Restablecer de fabrica permisos

**Comment by divyamlathiya** [comment]
Spanish:
Para restablecer permisos de fábrica en un dispositivo, ve a "Configuración" > "Restablecer" > "Restablecer ajustes de fábrica" o "Restaurar valores predeterminados". Esto borra configuraciones personalizadas y permisos.

English:
To reset factory permissions on a device, go to "Settings" > "Reset" > "Factory Reset" or "Restore Default Settings." This will erase custom configurations and permissions.
